understanding/non-text-contrast.html

```diff
@@ -108,395 +102,279 @@ <h1><span class="standalone-resource__type-of-guidance">Understanding
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   				
-                  <dt>Goal</dt>
-                  <dd>Important visual information meets the same minimum contrast required for larger text.</dd>
+                  <dt>目標</dt>
+                  <dd>重要な視覚的情報が、大きな文字に対して求められているのと同様の最低限のコントラストを満たすようにする。</dd>
                   				
-                  <dt>What to do</dt>
-                  <dd>Ensure meaningful visual cues achieve 3:1 against the background.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>意味を成す視覚的な手がかりが、背景色に対して 3:1 のコントラスト比になるようにする。</dd>
                   				
-                  <dt>Why it's important</dt>
-                  <dd>Some people cannot see elements with low contrast.</dd>
+                  <dt>なぜそれが重要か</dt>
+                  <dd>コントラストが低い要素を見ることができない人がいる。</dd>
                   			
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The intent of this Success Criterion is to ensure that user interface components (i.e.,
-                  controls) and meaningful graphics are distinguishable by people with moderately low
-                  vision. The requirements and rationale are similar to those for large text in <a href="contrast-minimum">1.4.3 Contrast (Minimum)</a>. Note that this requirement does not apply to <em>inactive</em> user interface components.
+               <h2>意図</h2>
+               <p>この達成基準の意図は、アクティブなユーザインタフェース コンポーネント (つまり、コントロール) 及び意味のあるグラフィックが中程度のロービジョンの人によって識別できるようすることである。この必須要件の根拠は、<a href="contrast-minimum">1.4.3 コントラスト (最低限)</a> の大きな文字のものと同様である。この要件は、<em>非アクティブな</em>ユーザインタフェース コンポーネントには適用されないことに注意されたい。
```

Comment:
- diffを取るとactiveは削除されているので
- ついでにrequirementsはほかの段落では単に要件としているようなのであわせます


```suggestion
               <p>この達成基準の意図は、ユーザインタフェース コンポーネント (つまり、コントロール) 及び意味のあるグラフィックが中程度のロービジョンの人によって識別できるようすることである。この要件の根拠は、<a href="contrast-minimum">1.4.3 コントラスト (最低限)</a> の大きな文字のものと同様である。この要件は、<em>非アクティブな</em>ユーザインタフェース コンポーネントには適用されないことに注意されたい。
```